Automattic/sensei
# Add notification when layouts are not available

Resolves #7391 

## Proposed Changes
* Display warning when there are no available layouts.

## Testing Instructions

1. Install and activate Divi. Enable Divi Builder for course and lesson post types.
2. Go to Courses and create a new course.
3. Make sure you see "No layouts available for this theme." on the layouts step.
4. Go to Lessons and create a new lesson.
5. Make sure you see "No layouts available for this theme. on the layouts step.
6. Now, activate a few other themes (Course, 2024, Astra) and check you don't see the warning when you see layouts.

## New/Updated Hooks

* `sensei.editorWizard.noLayoutsWarning` - Filter no layouts message.

## Pre-Merge Checklist
